cql3/statements/create_keyspace_statement.cc

```diff
@@ -240,7 +240,12 @@ std::vector<sstring> check_against_restricted_replication_strategies(
     // these are checked and reported elsewhere.
     for (auto opt : attrs.get_replication_options()) {
         try {
-            auto rf = std::stol(opt.second);
+            long rf = 0;
+            try {
+                rf = locator::get_replication_factor(opt.second);
+            } catch (const exceptions::configuration_exception&) {
```

Comment:
why is this exception ignored? (in [Support replication factor rack list for tablet-based keyspaces](https://github.com/scylladb/scylladb/pull/25625/commits/401f0905e49b94f4226072f3a1957c7bd5bcc4e0#top))

Reply:
This whole approach is a hack and it's begging for refactoring. But out of scope.